src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,73 @@
+package calculator;
+
+import java.util.LinkedList;
+import type.MessageFactory;
+
+class StringCalculator {
+    private final CalculationElement<Double> operands;
+    private final CalculationElement<String> operators;
+
+    public static StringCalculator create() {
+        return new StringCalculator();
+    }
+
+    private StringCalculator() {
+        this(CalculationElement.from(new LinkedList<>()),
+                CalculationElement.from(new LinkedList<>()));
+    }
+
+    private StringCalculator(CalculationElement<Double> operands, CalculationElement<String> operators) {
+        this.operands = operands;
+        this.operators = operators;
+    }
```

Comment:
```
class A {
    상수(static final) 또는 클래스 변수
    인스턴스 변수
    생성자
    팩토리 메소드
    메소드
    기본 메소드 (equals, hashCode, toString)
}
```

```suggestion
    private StringCalculator() {
        this(CalculationElement.from(new LinkedList<>()),
                CalculationElement.from(new LinkedList<>()));
    }

    private StringCalculator(CalculationElement<Double> operands, CalculationElement<String> operators) {
        this.operands = operands;
        this.operators = operators;
    }

    public static StringCalculator create() {
        return new StringCalculator();
    }
```

클래스 구현 순서는 다음과 같습니다!

Reply:
놓쳤네용 ㅜ.ㅜ